pyzabbix/sender.py

```diff
@@ -117,7 +117,13 @@ def __init__(self, host, key, value, clock=None):
         self.key = str(key)
         self.value = str(value)
         if clock:
-            self.clock = clock
+            if isinstance(clock, float):
```

Comment:
Ok, so, if you really wanna check the type, it's can something like:
```python
if isinstance(clock, (float, int, None)):
    self.clock = int(clock)
else: 
    raise Exception('blah blah')
```

I think perhaps rising exception is a right behaviour instead of warnings module. Also no need use warning in this case it's better add standard logger.
```python
self.log.warn('blah-blah')
```

`warnings` module is good for specific type of warnings, which actually might won't be shown by default in python. Like deprecation warnings.